src/mesido/gas_physics_mixin.py

```diff
@@ -246,15 +253,30 @@ def _get_min_bound(bound):
 
         self.__maximum_total_head_loss = self.__get_maximum_total_head_loss()
 
+        if options["gas_storage_discharge_variables"]:
+            for storage in self.energy_system_components.get("gas_storage", []):
+                bound_storage_q = self.bounds()[f"{storage}.GasIn.Q"][0]
+                var_name = f"{storage}__Q_discharge"
+                self.__gas_storage_discharge_map[storage] = var_name
+                self.__gas_storage_discharge_var[var_name] = ca.MX.sym(var_name)
+                self.__gas_storage_discharge_bounds[var_name] = (0, -bound_storage_q)
+                self.__gas_storage_discharge_nominals[var_name] = self.variable_nominal(
+                    f"{storage}.GasIn.Q"
+                )
+
     def energy_system_options(self):
         r"""
         Returns a dictionary of milp network specific options.
 
+        gas_storage_discharge_variables: creates separate variables for the discharge of the
+        gas_storages, only required when using the multicommodity simulator as this requires
+        separate variables to create goals with.
         """
 
         options = self._gn_head_loss_class.head_loss_network_options()
 
         options["minimum_pressure_far_point"] = 1.0
+        options["gas_storage_discharge_variables"] = False
```

Comment:
Should this not rather be set in -->>         self.gas_network_settings = {
            "network_type": NetworkSettings.NETWORK_TYPE_GAS,
            "maximum_velocity": 15.0, ......................................

Reply:
Is also possible, as the name is different for gas and electricity, I thought I could just aswell be set in the overall options.